client-app/shared/account/components/product-card-reorder.vue

```diff
@@ -19,25 +19,35 @@
             v-if="(props.productItem.quantity! > props.productItem.availabilityData?.availableQuantity) && !isInputDisabled"
           >
             <i class="fas fa-exclamation-circle text-[color:var(--color-primary)] self-start mt-1"></i>
-            <span class="text-xs text-gray-400">
-              The number of items in the order was
-              <span class="text-gray-500"
-                >reduced from {{ props.productItem.quantity }} to
-                {{ props.productItem.availabilityData?.availableQuantity }}</span
-              >
-              due to stock restrictions
+            <span
+              class="text-xs text-gray-400"
+              v-html="
+                $t('shared.account.reorder_info_popup.product_card.reduced_message', [
```

Comment:
Why all the keys in this file are inside `reorder_info_popup` namespace? Possible wrong namespace.

Reply:
Becouse it is a child component for reorder-info popup